README.md

```diff
@@ -37,7 +37,8 @@ These libraries contain helper functions and other tools valuable to blockchain
 
 #### Access Control and Security
 
-- [Ownership](./libs/ownership/) is used to apply restrictions on functions such that only a single user may call them.
+- [Ownership](./libs/ownership/) is used to apply restrictions on functions such that only a **single** user may call them.
+- [Admin](./libs/admin/) is used to apply restrictions on functions such that only a select few users may call them.
```

Comment:
```suggestion
- [Admin](./libs/admin/) is used to apply restrictions on functions such that only a select few users may call them.
- [Ownership](./libs/ownership/) is used to apply restrictions on functions such that only a **single** user may call them.
```

Reply:
It doesn't make any sense to list an extension before it's dependency.

Reply:
Keeping things organized by lexicographical ordering. No one except the author is thinking about dependency ordering here. From the user's perspective, and UX, it makes sense.